### ISSUE TITLE
[data request] DRIVE: Digital Retinal Images for Vessel Extraction

### ISSUE THREAD

**Issue by zdz1130** [tensorflow/datasets#2384]
* Name of dataset: DRIVE: Digital Retinal Images for Vessel Extraction
* URL of dataset: https://www.dropbox.com/sh/z4hbbzqai0ilqht/AAARqnQhjq3wQcSVFNR__6xNa?dl=0
* License of dataset: public
* Short description of dataset and use case(s): The photographs for the DRIVE database were obtained from a diabetic retinopathy screening program in The Netherlands. The screening population consisted of 400 diabetic subjects between 25-90 years of age. Forty photographs have been randomly selected, 33 do not show any sign of diabetic retinopathy and 7 show signs of mild early diabetic retinopathy.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by rohit11544** [comment]
can I add this dataset ??

**Comment by zdz1130** [comment]
> can I add this dataset ??

@rohit11544 sure. you are more than welcome to contribute.

**Comment by rohit11544** [comment]
It may take more time for me to complete this because of my collage works so this issue is open for everyone. This research paper may help you to get started with the data set. Link :  https://github.com/rohit9934/DRIVE-Digital-Retinal-Images-for-Vessel-Extraction/blob/master/DRIVE/DRIVE%20(1).pdf

**Comment by humblef-oo-l** [comment]
I am not getting 2 things. Please help me, in training set, I have got the manual of images and the mask, but why am I not getting any manual for testing. The manual is those white arteries on the black background for images.
